examples/cluster.yml

```diff
@@ -16,6 +16,7 @@ spec:
     etcd:
       domain: "k8s.cluster.giantswarm.io"
       prefix: "example-cluster"
+      port: 2379
```

Comment:
So what is the idea conceptually? Exposing etcd on its well known standard port via domain using TLS? I remember we defined two ports being open for etcd. Will one be additionally open internally as well? Would like to have a clear picture to get that straight.